### PR TITLE
[CDAP-15389] Fixes markdown tables to be standard representation

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Fixes tables, code snippets to adhere to github flavored markdown to be able to apply styles in UI.

### CODE REVIEW

**Review by albertshau — COMMENTED** [review]
there is a wrangler submodule change that snuck in but otherwise lgtm

File: core-plugins/docs/JavaScript-transform.md [hunk 1]
```diff
@@ -68,64 +70,62 @@ Currently supports ``KeyValueTable``.
...
-            }",
-            "lookup": "{
-                \"tables\":{
-                    \"purchases\":{
-                        \"type\":\"DATASET\",
-                        \"datasetProperties\":{
-                            \"dataset_argument1\":\"foo\",
-                            \"dataset_argument2\":\"bar\"
-                        }
+```js
```
Comment (elfenheart) [inline core-plugins/docs/JavaScript-transform.md:73]: this is not `json`?

File: core-plugins/docs/Validator-transform.md [hunk 2]
```diff
@@ -138,34 +142,36 @@ field is greater than 1MB.
...
-                                     isValid = false; errMsg = \"content length >1MB\"; errCode = 10;
-                                     metrics.count(\"invalid.body.size\", 1);
-                                  }
-                                  if (!isValid) {
-                                    logger.warn(\"Validation failed for record {}\", input);
-                                  }
-                                  return {'isValid': isValid, 'errorCode': errCode, 'errorMsg': errMsg};
-                                };"
-        }
+```javascript
```
Comment (elfenheart) [inline core-plugins/docs/Validator-transform.md]: should this be js or javascript? or are they interchangeable?

File: core-plugins/docs/JavaScript-transform.md (hunk 1 above)
Comment (ajainarayanan) [reply core-plugins/docs/JavaScript-transform.md:73]: You are right it should be json. But having this a `js` helps in highlighting the code as part of the JSON + js also highlights json syntax as well.

File: core-plugins/docs/Validator-transform.md (hunk 2 above)
Comment (ajainarayanan) [reply core-plugins/docs/Validator-transform.md]: It is [interchangeable](https://prismjs.com/#supported-languages). We can either user `js` or `javascript` 
**EDIT:** Have changed to `js` for consistency.

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Pull request merged**